### PR TITLE
Feature/sof 4430 - May Webinar: AutoML for Chemistry/Materials

### PR DESCRIPTION
**Pull request by AcylSilane**
AutoML and Hyperparameter Optimization

### CODE REVIEW

File: examples/machine_learning/Classification_Bitter.py [hunk 1]
```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
```
Comment (timurbazhirov) [inline examples/machine_learning/Classification_Bitter.py:1]: `Classification_of_Bitterness_XGBoost.ipynb` is a better name

File: examples/machine_learning/Perovskite_Processing.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Predicting Perovskite Volumes
```
Comment (timurbazhirov) [inline examples/machine_learning/Perovskite_Processing.py:4]: `Prediction_of_Perovskite_Unit_Cell_Volume.ipynb` is a better name

File: examples/machine_learning/Classification_Bitter.py [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+# coding: utf-8
+
+# # Bitterness Classification With XGBoost
```
Comment (timurbazhirov) [inline examples/machine_learning/Classification_Bitter.py:4]: We need to add a README file in `example/assets` and explain what the "bitter-sweet.csv` file is. The name is ambiguous again.

File: examples/machine_learning/Classification_Bitter.py (hunk 1 above)
Comment (AcylSilane) [reply examples/machine_learning/Classification_Bitter.py:1]: Renamed

File: examples/machine_learning/Perovskite_Processing.py (hunk 2 above)
Comment (AcylSilane) [reply examples/machine_learning/Perovskite_Processing.py:4]: Renamed

File: examples/machine_learning/Classification_Bitter.py (hunk 3 above)
Comment (AcylSilane) [reply examples/machine_learning/Classification_Bitter.py:4]: Added

File: .gitattributes [hunk 4]
```diff
@@ -2,5 +2,3 @@
 *.csv filter=lfs diff=lfs merge=lfs -text
 *.vasp filter=lfs diff=lfs merge=lfs -text
 *.poscar filter=lfs diff=lfs merge=lfs -text
-examples/assets/* filter=lfs diff=lfs merge=lfs -text
-"examples/assets/mp-978534.poscar" filter=lfs diff=lfs merge=lfs -text
```
Comment (timurbazhirov) [inline .gitattributes:6]: What happened here??

Comment (AcylSilane) [reply .gitattributes:6]: The assets folder readme does not display to the user in Git if it is an LFS pointer. The only files in that directory are csv and poscar files, anyway.

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**